gdx/src/com/badlogic/gdx/math/collision/Ray.java

```diff
@@ -104,7 +104,7 @@ public Ray set (float x, float y, float z, float dx, float dy, float dz) {
 	 * @return This ray for chaining */
 	public Ray set (Ray ray) {
 		this.origin.set(ray.origin);
-		this.direction.set(ray.direction);
+		this.direction.set(ray.direction).nor();

```

Comment:
It's minor but there's no need to normalize as the passed ray will have its direction already normalized

Reply:
ye, i got this interogation of the utility in this case. 
I add the nor() for be similar with other method.